src/rules/property-no-unknown/__tests__/index.js

```diff
@@ -0,0 +1,81 @@
+import { testRule } from "../../../testUtils"
+import rules from "../../../rules"
+import { ruleName, messages } from ".."
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+
+  accept: [ {
+    code: ".foo { color: green; }",
+  }, {
+    code: ".foo { fill: black; }",
+  }, {
+    code: ".foo { -webkit-align-self: center; }",
+  }, {
+    code: ".foo { align-self: center; }",
+  }, {
+    code: ".foo { --bg-color: white; }",
+    description: "ignore standard CSS variables",
+  } ],
+
+  reject: [ {
+    code: ".foo { colr: blue; }",
+    message: messages.rejected("colr"),
+    line: 1,
+    column: 8,
+  }, {
+    code: ".foo { -moz-align-self: center; }",
```

Comment:
Looking through the [tests checklist](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#checklist), can we also get:

1. A test which is rejected on a different line and column to two tests above e.g. `.foo {\n  colr: blue;\n}`
2. A passing and rejected case insensitivity tests. e.g. `.foo { COLOR: blue; }` and `.foo { COLR: blue; }` 

Other than that, it looks like you're succinctly covered off the applicable parts of the checklist.